sui-execution/latest/sui-verifier/Cargo.toml

```diff
@@ -15,6 +15,7 @@ move-core-types.workspace = true
 move-vm-config.workspace = true
 move-abstract-stack.workspace = true
 
+move-abstract-interpreter = { path = "../../../external-crates/move/crates/move-abstract-interpreter" }
```

Comment:
I would expect something similar in the legacy cuts, no?

Reply:
Only the new/latest cuts have this crate. The previous ones have it already through the versioned bytecode verifier